data/init.sql

```diff
@@ -137,34 +137,34 @@ CREATE TABLE IF NOT EXISTS courses (
 	updated_at TIMESTAMP
 ); 
 
-CREATE TABLE IF NOT EXISTS course_offerings(
+CREATE TABLE IF NOT EXISTS lectures (
 	id INT(11) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
 	course_id INT(11) UNSIGNED,
 	FOREIGN KEY(course_id) REFERENCES courses(id) ON DELETE CASCADE,
-	course_offering_code VARCHAR(60),
-	course_offering_name VARCHAR(60),
+	lecture_code VARCHAR(60),
+	lecture_name VARCHAR(60),
 	created_at TIMESTAMP,
 	updated_at TIMESTAMP
 );
 
-CREATE TABLE IF NOT EXISTS course_sections(
+CREATE TABLE IF NOT EXISTS sections (
 	id INT(11) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
-	offering_id INT(11) UNSIGNED,
-	FOREIGN KEY(offering_id) REFERENCES course_offerings(id) ON DELETE CASCADE,
-	course_section_code VARCHAR(60),
-	course_section_name VARCHAR(60),
+	lecture_id INT(11) UNSIGNED,
+	FOREIGN KEY(lecture_id) REFERENCES lectures(id) ON DELETE CASCADE,
+	section_code VARCHAR(60),
```

Comment:
same here, do we really need a code and a name?

Reply:
wait why do we need both lectures and sections? wouldnt it make sense to simply have courses->lectures->tutorials?

Reply:
well its just a name, lets call them either one and get this done with